### PR TITLE
🍱 Better French translation

### PR DESCRIPTION
**Pull request by FlipFloop**
**Changes proposed in this pull request:**

- Better french translations.

**Help wanted:** (optional, e.g. a request for GUI revamp)

- The package-lock.json change is non-existent, but appears.

**Ping @CosmoMyzrailGorynych**
**Ping @markmehere**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -24298,4 +24298,4 @@
       }
     }
   }
-}
+}
```
Comment (CosmoMyzrailGorynych) [inline package-lock.json:24301]: What the heck happens here anyways <_>

**Review by CosmoMyzrailGorynych — APPROVED** [review]

**Comment by CosmoMyzrailGorynych** [comment]
image
<_> seems that change is not that non-existent for git

**Comment by CosmoMyzrailGorynych** [comment]
Okay, I've merged it manually

**Comment by FlipFloop** [comment]
Lovely, time to do real stuff

**Pull request merged**